source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -71,62 +161,98 @@ To authenticate with ECR, you will need to set:
-      fi
-```
-#### Deploy to Kubernetes
+version: 2.1
 
-We provide a docker image that simplifies the CircleCI configuration by encapsulating the authentication process in a script. For example, given a configured `DEVELOPMENT` environment (see the section on environment variables above):
+workflows:
+  test-env-vars:
+    jobs:
+      - build:
+          context: <CONTEXT>
 
-```yaml
-deploy_development:
-  docker:
-    - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
-  steps:
+jobs:
+  build:
+    docker:
+    - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci-internal
+      environment:
+        IMAGE_URI: ${ECR_ENDPOINT}/<REPO-NAME>
+    steps:
     - checkout
-    - deploy:
-        name: Helm deployment
+    - setup_remote_docker
+    - run:
+        name: Build application Docker image and push to ECR
+        command: |
+          $(aws ecr get-login --no-include-email)  
+          docker build -t ${ECR_ENDPOINT}/<REPO-NAME> rails-app
+          docker push ${ECR_ENDPOINT}/<REPO-NAME>
+    - run:
+        name: Authenticate with cluster
+        command: |
+          echo -n ${K8S_CLUSTER_CERT} | base64 -d > ./ca.crt
+          kubectl config set-cluster ${K8S_CLUSTER_NAME} --certificate-authority=./ca.crt --server=https://api.${K8S_CLUSTER_NAME}
+          kubectl config set-credentials circleci --token=${K8S_TOKEN}
+          kubectl config set-context ${K8S_CLUSTER_NAME} --cluster=${K8S_CLUSTER_NAME} --user=circleci --namespace=${K8S_NAMESPACE}
+          kubectl config use-context ${K8S_CLUSTER_NAME}
+    - run:
+        name: Install demo helm chart
         command: |
-          setup-kube-auth
-          kubectl config use-context development
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            helm upgrade ${APPLICATON_DEPLOY_NAME} ./helm_deploy/django-app/. \
-                          --install \
-                          --tiller-namespace=${NON_PROD_NS} \
-                          --namespace=${NON_PROD_NS} \
-                          --set image.repository="${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${CIRCLE_PROJECT_REPONAME}" \
-                          --set image.tag="${CIRCLE_SHA1}" \
-                          --set deploy.host="${APPLICATION_HOST_URL}" \
-                          --set replicaCount="4" \
-                          --debug
-          fi
+          if [ "${CIRCLE_BRANCH}" == "master" ]; th en
+            helm upgrade demo helm_deploy --install --namespace=${K8S_NAMESPACE}
+          fi 
+          kubectl --namespace=${K8S_NAMESPACE} get pods
 ```
 
-[deploy-helm]: /documentation/deploying-an-app/app-deploy-helm.html#deploying-an-application-to-the-cloud-platform-with-helm
+In the above config.yml replace CONTEXT with the name of the context you created. 
+Also replace REPO-NAME with the name of your ECR repo proceeding the tag name. For example if my repo name is my-repo/test and my image tag is rails then I would replace
```

Comment:
I think "...excluding the tag name" is clearer than "...preceding the tag name"

Also, it would be great if you could use examples that look more like what the user will be seeing, at this point. e.g.
```
  754256621582.dkr.ecr.eu-west-2.amazonaws.com/myteam/myrepo:rails
```

Reply:
done

Reply:
Thanks for this. I think the paragraph would be easier to read with a bit more whitespace, and some backtick formatting. e.g.

---

In the above config.yml replace <CONTEXT> with the name of the context you created. Also replace <REPO-NAME> with the name of your ECR repo excluding the tag name. 

For example if my repo name is `myteam/myrepo` and my image tag is `rails` then I would replace <REPO-NAME> with `myteam/myrepo:rails` with the full repo URL being:

```
754256621582.dkr.ecr.eu-west-2.amazonaws.com/myteam/myrepo:rails
```